en/flight_controller/pixhawk6c_mini.md

```diff
@@ -0,0 +1,209 @@
+# Holybro Pixhawk 6C Mini
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://holybro.com/) for hardware support or compliance issues.
+:::
+
```

Comment:
@vincentpoont2  all this marketing stuff is pretty enough, but it duplicates the 6C docs - so not clear why you would take the 6c mini (Yes, I understand the reason, but this does not document it). I'm going to merge this but you might want to tidy as a post process.